src/app/launcher/create-app/project-summary-createapp-step/project-summary-createapp-step.component.ts

```diff
@@ -93,8 +93,9 @@ export class ProjectSummaryCreateappStepComponent extends LauncherStep implement
    * Set up this application
    */
   setup(): void {
-    this.subscriptions.push(this.projectSummaryService.setup(this.launcherComponent.summary).subscribe((val) => {
-      if (val === true) {
+    this.subscriptions.push(this.projectSummaryService.setup(this.launcherComponent.summary).subscribe((val: any) => {
+      if (val && val['uuid_link']) {
+        this.launcherComponent.summary.uuidLink = val['uuid_link'];
```

Comment:
Please do not store anything in the summary which is not displayed in the summary section itself.

Reply:
sure. i will work on it.